examples/prom/mixed_nonlinear_diffusion.cpp

```diff
@@ -31,8 +31,9 @@
 //               Analytic test (reproductive)
 //               mpirun -n 1 ./mixed_nonlinear_diffusion -offline
 //               mpirun -n 1 ./mixed_nonlinear_diffusion -merge -ns 1
-//               mpirun -n 1 ./mixed_nonlinear_diffusion -online -rrdim 8 -rwdim 8 -nldim 20 -nsdim 20
-//               mpirun -n 1 ./mixed_nonlinear_diffusion -online -rrdim 8 -rwdim 8 -nldim 20 -nsdim 20 -sopt
+//               mpirun -n 1 ./mixed_nonlinear_diffusion -online -rrdim 8 -rwdim 8 -nldim 20 -nsdim 20 -hrtype deim
+//               mpirun -n 1 ./mixed_nonlinear_diffusion -online -rrdim 8 -rwdim 8 -nldim 20 -nsdim 20 -hrtype qdeim
+//               mpirun -n 1 ./mixed_nonlinear_diffusion -online -rrdim 8 -rwdim 8 -nldim 20 -nsdim 20 -hrtype sopt
 //               mpirun -n 1 ./mixed_nonlinear_diffusion -online -rrdim 8 -rwdim 8 -nldim 20 -nsdim 20 -ns 1 -eqp
 //
 //               Relative l2 error of ROM solution at final timestep using DEIM sampling: 1.096776797994166e-08
```

Comment:
I believe this result corresponds to `-hrtype deim`? Perhaps we should report the example results for Q-DEIM and S-OPT here and in the libROM webpage.

Reply:
My first thought was not to add it, because the QDEIM result is almost the same as DEIM. For completeness, I went ahead and added it. I agree that the webpage should be updated as well, to let users know about the QDEIM option.
I actually updated all the run times. Reporting run times is troublesome, because they vary even on the same machine. When a new result is added, all runs have to be updated for a fair comparison.